charts/trino/templates/service.yaml

```diff
@@ -7,6 +7,10 @@ metadata:
     chart: {{ template "trino.chart" . }}
     release: {{ .Release.Name }}
     heritage: {{ .Release.Service }}
+{{- if .Values.service.annotations }}
```

Comment:
Could you add this with an empty default to the `values.yaml` file? Otherwise, users might not know that specifying this is possible, without looking into this template.

Reply:
@nineinchnick I've been struggling with adding the following annotations for my service type "LoadBalancer":
```
    service.beta.kubernetes.io/aws-load-balancer-type: nlb
    service.beta.kubernetes.io/aws-load-balancer-nlb-target-type: instance
    service.beta.kubernetes.io/aws-load-balancer-scheme: internal
    service.beta.kubernetes.io/aws-load-balancer-internal: "true"
```

Is this supported yet, if yes/no how to achieve this? Please help me out.

Reply:
This PR has not been merged yet.

Reply:
It is super basic feature they should've merged this already.
Any takeoffs from this?

Reply:
This has nothing to do with this pr. Please ask questions like this on slack.

Reply:
Cool, thanks.

Reply:
@nineinchnick 
Actually, I never got an answer to my question on slack, somehow I figured that out on my own.

Also, if possible can you please help me with the following?

I'm trying to use the "[Trino Official Helm Chart](https://github.com/trinodb/charts/tree/main/charts/trino)", I've it all set up and working properly in a k8s cluster with "HTTPS" and "PASSWORD" based authentication "on".

The only thing I can't figure out is, how to pass a "password.db" file instead of "passing a string" as "username:hashed-bcrypt-password-string".

Reply:
Please stop hijacking this comment to ask questions not related to this pr. I saw your questions on slack but I don't know the answer. If anyone else does, they'll reply there.

Reply:
Sorry.